services/core-api/app/api/mines/permits/permit_amendment/models/permit_amendment.py

```diff
@@ -37,6 +37,7 @@ class PermitAmendment(SoftDeleteMixin, AuditMixin, Base):
     description = db.Column(db.String)
     issuing_inspector_title = db.Column(db.String)
     regional_office = db.Column(db.String)
+    preamble_text = db.Column(db.String, nullable=False, server_default=FetchedValue())
 
```

Comment:
nullable=True, and I don't think `server_default` is necessary here

Reply:
Great call, it was defined like that when the solution included the preamble_text to be generated from the database which is not longer the case.